hbase-client/src/main/java/org/apache/hadoop/hbase/client/RawAsyncTableImpl.java

```diff
@@ -218,50 +222,62 @@ private static Result toResult(HBaseRpcController controller, MutateResponse res
 
   @Override
   public CompletableFuture<Result> get(Get get) {
-    return timelineConsistentRead(conn.getLocator(), tableName, get, get.getRow(),
-      RegionLocateType.CURRENT, replicaId -> get(get, replicaId), readRpcTimeoutNs,
-      conn.connConf.getPrimaryCallTimeoutNs(), retryTimer, conn.getConnectionMetrics());
+    return tracedFuture(
```

Comment:
For AsyncTable, its get() method is implemented by RawAsyncTable's get() method. So here will trace two times? But AsyncTable's get() method's cost time should be very close with RawAsyncTable's get() method. Do we need to trace them both? And I thought the "trace tool" may be more useful for RPC call. For local call stacktrace, profiler tool is more useful?

Reply:
So you mean we just do not need to add trace support for methods declared in AsyncTable interface, as finally they will call other methods which should already been traced?

Reply:
Good to keep the trace macro at first.... entrance and exit, yeah. Simple. Later, when demand, can add trace across internal 'systems'... is the time being taken adding to memstore or appending WAL -- that kinda thing.

So, yeah, entrance/exit on AsyncTable Interface methods would be good place to start I'd think.